### PR TITLE
ENG-1388 jira Docs

### PR DESCRIPTION
**Pull request by favnc**

### CODE REVIEW

File: jira/webhook-integration.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          - type: xlrelease.StringValue
+            targetProperty: releaseTitle
+            value: Starting release from jira webhooks for ${event.issue.key}
+          - type: xlrelease.StringValue
+            targetProperty: variables[description]
+            value: ${event.issue.fields.issuetype.description}
+          - type: xlrelease.ListStringValue
+            targetProperty: tags
+            value:
+              - ${ event.issue.key}
```
Comment (bstevens736) [inline jira/webhook-integration.yaml]: should there be a space between the brace and "event"?

File: jira/webhook-integration.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          - type: xlrelease.StringValue
+            targetProperty: variables[release_name]
+            value: ${event.issue.key}
+          - type: xlrelease.ListStringValue
+            targetProperty: variables[items]
+            value:
+              - ${event.changelog.items}
+          - type: xlrelease.StringValue
+            targetProperty: variables[project_name]
+            value: '{$event.issue.fields.project.name}'
```
Comment (bstevens736) [inline jira/webhook-integration.yaml]: This looks odd

**Review by bstevens736 — COMMENTED** [review]
The YAML in its current state fails when applying with:

Unexpected response: The reference with name [Jira Webhooks Endpoint] in the field [eventSource] should be created first before trying to use it.

The endpoint is missing in the yaml.  If you generated from config then there must be a bug in the cli or maybe you are using an old version?

File: jira/webhook-integration.yaml (hunk 2 above)
Comment (bstevens736) [reply jira/webhook-integration.yaml]: ignore this if it works...just thought it was odd how this value is formatted differently than all others.  Why is this in single quotes and the $ inside the brace.

Comment (favnc) [reply jira/webhook-integration.yaml]: Sure, @bstevens736 I generated this running xl cli

File: jira/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    }
+  }
+}
+``` 
+
+#### Data mapping
+
+Fields from the incoming JSON event and Release variables can be mapped directly to release properties like:
+
+![screenshot of ReleaseformTemplate](screenshots/release_prop.png)
```
Comment (bstevens736) [inline jira/README.md]: this screenshot shows the bad variable mapping that you fixed in the yaml.  Probably should generate a new screenshot

File: jira/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+
+After the trigger has fired and the release was created, the properties and variables of the release will contain the data from the incoming Jira event: 
+
+![screenshot of TemplateAfterRelease](screenshots/release_complete.png)
```
Comment (bstevens736) [inline jira/README.md]: this screenshot shows the bad variable mapping that you fixed in the yaml.  Probably should generate a new screenshot

**Review by bstevens736 — COMMENTED** [review]
Besides the two screenshots everything else looks go to me

File: jira/README.md (hunk 3 above)
Comment (favnc) [reply jira/README.md]: done

File: jira/webhook-integration.yaml [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+              - ${event.issue.key}
+              - ${event.webhookEvent}
+              - ${event.timestamp}
+          - type: xlrelease.StringValue
+            targetProperty: variables[release_name]
+            value: ${event.issue.key}
+          - type: xlrelease.ListStringValue
+            targetProperty: variables[items]
+            value:
+              - ${event.changelog.items}
```
Comment (bdevic) [inline jira/webhook-integration.yaml]: Here looks like we are binding a list as a string item of the items list?

**Pull request merged**